src/components/SearchBox.tsx

```diff
@@ -58,6 +58,32 @@ const SearchBox = ({ setSearchResults }: SearchBoxProps) => {
     }
   };
 
+  const getRandomArtist = async () => {
+    try {
+      const { data } = await axios.get(RANDOM_ARTIST_API_URL);
```

Comment:
There is a huge `any` party in this function, would be awesome to see some return types defined